samples/net/echo_server/src/tcp.c

```diff
@@ -191,9 +191,7 @@ void start_tcp(void)
 
 void stop_tcp(void)
 {
-#if defined(CONFIG_NET_APP_TLS)
-	net_app_server_tls_disable(&tcp);
```

Comment:
this one still being mentioned in the docs:

doc/subsystems/networking/net-app-api.rst::cpp:func:`net_app_server_tls_disable()`.